src/integrationTest/java/com/mongodb/kafka/connect/embedded/EmbeddedKafka.java

```diff
@@ -433,6 +425,7 @@ public void deleteTopicsAndWait(final Duration duration, final String... topics)
       try {
         broker.deleteTopic(topic);
       } catch (final UnknownTopicOrPartitionException e) {
+        // Ignore
```

Comment:
Do we want to ignore exceptions or at least maybe log them?

Reply:
This is just testing infrastructure as part of the clean up, if the topic doesn't exist then its fine.

I added the comment to make intellij happy.